src/utils/constants.ts

```diff
@@ -0,0 +1,8 @@
+export const helperLinesNumber = 3
+
+export const defaultOponentTeamColor = '#000000'
```

Comment:
This should go in the [appConfig](https://github.com/NordicPlayground/robot-war-app/blob/4386aa4c837775d8d839dee203e3f76660122cde/src/hooks/useAppConfig.tsx) hook, so we have one place for application configuration.